### PR TITLE
Update Hugo version to avail of new Blackfriday options

### LINKED ISSUE #481: [WEBSITE] Update Hugo version
## Issue description

There are security issues that have been fixed in latest version of Blackfriday, and that version will be available in Hugo v0.68 (see https://github.com/gohugoio/hugo/issues/7039). We are currently using Hugo version 0.55.

**Update: The fix has been released in 0.67.1**

## Problem location

n/a

## Suggestions for a fix

Update the Hugo version to 0.68 or later and ensure that the required Blackfriday options are turned on in our config.

### PR DESCRIPTION
**Pull request by alexearnshaw**
## Describe the changes

* Upgrade Hugo locally to 0.66.0
* Set Blackfriday as default parser and turn on the extra options:
```
nofollowLinks = true 
noreferrerLinks = true 
noopenerLinks = true 
```
* Set Hugo version in Netlify config to 0.66.0

Fixes #481 

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [x] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [x] You have applied a production label if this needs a manual push to Zoomin
* [x] You have added it to the correct GitHub project
* [x] You have added it to the correct Sprint milestone
* [x] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [x] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [x] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

**Comment by alexearnshaw** [comment]
@andreamussap This is ready for review

File: config.toml [hunk 1]
```diff
@@ -32,12 +32,29 @@ pygmentsStyle = "tango"
...
+  latexDashes = true
+  nofollowLinks = true #this overrides the default setting of false
+  noreferrerLinks = true #this overrides the default setting of false
+  noopenerLinks = true #this overrides the default setting of false
+  
+  [markup.goldmark]
+    [markup.goldmark.renderer]
+      unsafe = true #this overrides the default setting of false
+
+## old configuration for BlackFriday markdown parser: https://github.com/russross/blackfriday
```
Comment (andreamussap) [inline config.toml:52]: @alexearnshaw why to keep this commented out in the file if we can always go back to the PR and see what was changed? Just thinking about letting the file as lean as possible ;)

**Comment by andreamussap** [comment]
> @andreamussap This is ready for review

I'm on it

File: content/en/docs/contribution_guidelines/maintain_customize.md [hunk 2]
```diff
@@ -73,6 +73,17 @@ The Docsy theme is installed in this site's git repo as a git submodule. To upda
     git submodule update --recursive
     ```
 
+## Update Hugo version
```
Comment (andreamussap) [inline content/en/docs/contribution_guidelines/maintain_customize.md:76]: **Update Hugo version x Upgrade Hugo** - could we make these headings more specific like:
- Update Hugo version in your site
- Update Hugo version on Netlify
- Upgrade Hugo on your local site
??
Also I miss the information on why to update/upgrade the Docsy theme and Hugo, we don't say this in the instructions, also it's not clear if this is a recommendation or mandaroty?

Comment (alexearnshaw) [reply content/en/docs/contribution_guidelines/maintain_customize.md:76]: Good feedback, I've added some why info and clarified the headings/instructions

**Comment by alexearnshaw** [comment]
@andreamussap This is ready for review again, thanks!

**Pull request merged**